doc/man1/s_server.pod

```diff
@@ -709,6 +711,15 @@ greater than or equal to 0.
 
 Accept early data where possible.
 
+=item B<-anti_replay>, B<-no_anti_replay>
+
+Switches on or off replay protection respectively. Normally, replay protection
```

Comment:
s/Normally, replay/Replay/

You already say "on by default"  below, so no need for "Normally,".

Reply:
The first sentence becomes:  "Switches replay protection on or off, respectively".